lametro/events.py

```diff
@@ -405,13 +409,13 @@ def scrape(self, window=None):
                 )
             else:
                 approved_minutes = self.find_approved_minutes(event)
-                if approved_minutes:
+                for minutes in approved_minutes:
                     e.add_document(
-                        note=approved_minutes["MatterAttachmentName"],
-                        url=approved_minutes["MatterAttachmentHyperlink"],
+                        note=minutes["MatterAttachmentName"],
+                        url=minutes["MatterAttachmentHyperlink"],
                         media_type="application/pdf",
                         date=self.to_utc_timestamp(
-                            approved_minutes["MatterAttachmentLastModifiedUtc"]
+                            minutes["MatterAttachmentLastModifiedUtc"]
```

Comment:
>Likewise, it assumes that an event will only ever have one minutes file. We'll need to revise that, too.

This should account for multiple minutes being returned from `find_approved_minutes`